src/Raven.Studio/wwwroot/Content/css/pages/storage-report.less

```diff
@@ -437,4 +448,28 @@
             margin-left: -4px;
         }
     }
+
+    .th-min-width {
```

Comment:
I think we should remove th from selectors. It will work even when you change this to div right?

Reply:
+ maybe we can define limited width on parent level and override single column with min-width? By doing this we don't have to repeat classes and we use cascading feature from CSS.

Reply:
I've made it a bit different, with the same effect. I've also added the `title` attribute to the URL in Name column, so the full name will be visible on hover.